api/pkg/transformer/pipeline/upi_postprocess_output_op.go

```diff
@@ -34,7 +35,10 @@ func (up *UPIPostprocessOutputOp) Execute(ctx context.Context, env *Environment)
 		return fmt.Errorf("not valid type %T", modelResponse)
 	}
 
-	copiedResponse := upiModelResponse
+	copiedResponse := &types.UPIPredictionResponse{}
+	if err := copier.CopyWithOption(copiedResponse, upiModelResponse, copier.Option{IgnoreEmpty: true, DeepCopy: true}); err != nil {
```

Comment:
Why do we copy in here?

Reply:
yeah, i think this one unnecessary since if we do modification in predictor response is still ok to be mutated since it won't be referred in another operation. Will remove the copy